notebooks/en/Prompt_Tuning_PEFT.ipynb

```diff
@@ -0,0 +1,7324 @@
+{
```

Comment:
<p>this generation is nsfw.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>I changed the input sentence to avoid this generation. </p>

Reply:
<p>Can you also update the output? </p>

Reply:
<p>Yes! For sure, I'm just executing the notebook again, and checking the responses obtained with different configurations. I will upload the notification in few minutes. </p>

Reply:
<p>This is the generation: 'There are two nice things that should matter to you: the weather and your health.' now. </p><p> </p><p>I checked with different configurations and all responses are sfw :-) </p>